### PR TITLE
Previewer fix

### PR DESCRIPTION
**Pull request by peteihis**
- Draw and render match in size
- The object fills the previewer better up
- Right button zoom is faster (speed feels closer to lateral move)
- Scroll zoom added

### CODE REVIEW

File: artofillusion/polymesh/MeshPreviewer.java [hunk 1]
```diff
@@ -168,7 +173,7 @@ public ObjectInfo getObject()
      * being displayed.
      */
 
-    public void setTexture(Texture tex, TextureMapping map) 
+    public void setTexture(Texture tex, TextureMapping map)
```
Comment (makiam) [inline artofillusion/polymesh/MeshPreviewer.java:176]: Looks like this method is not used as well as setMaterial

File: artofillusion/polymesh/MeshPreviewer.java [hunk 2]
```diff
@@ -259,11 +265,11 @@ private void drawObject(Graphics g)
 
     /** Rotate the object to show a specific side. */
 
-    private void changeView(int view) 
+    private void changeView(int view)
```
Comment (makiam) [inline artofillusion/polymesh/MeshPreviewer.java:268]: See no usages of this method too

File: artofillusion/polymesh/MeshPreviewer.java (hunk 1 above)
Comment (Sailsman63) [reply artofillusion/polymesh/MeshPreviewer.java:176]: Not relevant to this PR. Please open a separate issue. In particular, this is a public method - is the previewer perhaps being passed back to AOI core code somewhere, which could call this method?

Comment (peteihis) [reply artofillusion/polymesh/MeshPreviewer.java:176]: > is the previewer perhaps being passed back to AOI core code

I don't think so. It exists only on the mapping dialog/window. 

The PME MeshPreviewer code is copied from MaterialPreviewer of AoI, so there obviously is some (so far) unused residue code from the past. François has removed the parts, that used changeView() below, but I did not check how the rest of it works. Except for the parts, that I edited.. But I'm sure there's more to be found in PME. :)

**Pull request merged**